assets/src/scss/base/_icons.scss

```diff
@@ -9,50 +9,57 @@
 }
 
 a.standalone-link {
+  @include shared-link-styles;
+
+  display: inline-block;
+  width: fit-content;
+
   &::after {
     content: "";
+    display: inline-block;
+    background-color: currentColor;
     pointer-events: none;
     margin-inline-start: .2rem;
     height: .7rem;
     width: .7rem;
-    display: inline-block;
+    background-repeat: no-repeat;
     mask-image: url("../../images/chevron.svg");
-    mask-repeat: no-repeat;
     mask-size: contain;
+    mask-repeat: no-repeat;
     mask-position: center;
-    background-repeat: no-repeat;
-    background-color: currentColor;
   }
 }
 
 a.pdf-link {
   &::after {
     content: "";
     display: inline-block;
+    background-color: currentColor;
     height: 1rem;
     width: 1rem;
     margin-bottom: -2px;
     margin-inline-start: .2rem;
     margin-inline-end: .1rem;
-    background-color: currentColor;
     mask-image: url("../../images/file-pdf.svg");
     mask-repeat: no-repeat;
     mask-position: center;
   }
 }
 
 a.external-link {
+  @include shared-link-styles;
```

Comment:
Is this really needed? It seems it's duplicated:

<img width="354" alt="Screenshot 2023-10-23 at 14 23 34" src="https://github.com/greenpeace/planet4-master-theme/assets/6949075/e1c89e8d-c8ea-417f-bf72-aac92cd6c406">